### ISSUE TITLE
CREATE/DROP PROCEDURE display messages of functions

### ISSUE THREAD

**Issue by Xiang-Gu** [cockroachdb/cockroach#112419]
When I create or drop a procedure, the displayed message is create/drop functions.
```
demo@127.0.0.1:26257/demoapp/defaultdb> create procedure proc() language sql as 'SELECT 1';
CREATE FUNCTION

Time: 28ms total (execution 25ms / network 3ms)

demo@127.0.0.1:26257/demoapp/defaultdb> drop procedure proc;
DROP FUNCTION

Time: 65ms total (execution 63ms / network 1ms)
```

We should instead display `CREATE PROCEDURE` and `DROP PROCEDURE`, as we seem to intentionally distinguish between the two. For example, one couldn't use `DROP FUNCTION` to drop a procedure.

Jira issue: CRDB-32412

**Comment by whimSYZ** [comment]
Hi! I'm new to contributing and I wonder if I may take this issue?

**Comment by yuzefovich** [comment]
This should already be fixed by #112054
